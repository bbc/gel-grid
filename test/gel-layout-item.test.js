const sass = require('sass');

describe('gel-layout-item()', () => {
  it('creates a single layout group/row, this wraps individual layout items', async () => {
    const { css } = await sass.compileAsync('test/data/gel-layout-item/t1.scss', { quietDeps: true, logger: sass.Logger.silent });

    expect(css).toMatchSnapshot();
  });
});
