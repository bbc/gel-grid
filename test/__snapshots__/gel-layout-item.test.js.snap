// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`gel-layout-item() > creates a single layout group/row, this wraps individual layout items 1`] = `
".gel-layout-item {
  display: inline-block;
  padding-left: 8px;
  text-align: left;
  vertical-align: top;
  width: 100%;
}
@media (min-width: 37.5em) {
  .gel-layout-item {
    padding-left: 16px;
  }
}"
`;
